src/OrchardCore.Modules/Orchard.Liquid/Startup.cs

```diff
@@ -74,8 +56,7 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddLiquidFilter<ContainerFilter>("container");
             services.AddLiquidFilter<DisplayTextFilter>("display_text");
             services.AddLiquidFilter<DisplayUrlFilter>("display_url");
-            services.AddLiquidFilter<ContentUrlFilter>("content_url");
-            
+            services.AddLiquidFilter<ContentUrlFilter>("content_url");           
```

Comment:
Yes it is noted, i will do it when i will move things from `FluidView` to `O.Liquid`.